src/main/java/org/hydev/mcpm/server/crawlers/SpigetCrawler.java

```diff
@@ -44,7 +44,7 @@
     public SpigetCrawler(File dataDir)
     {
         this.dataDir = dataDir;
-        this.blacklist = new StoredHashMap<>(new File(dataDir, "crawler/spiget/blacklist.json"));
+        this.blacklist = new StoredHashMap<>(new File(dataDir, "crawler/spiget/blacklist.gs"));
```

Comment:
Why was the file name changed? 🤔

Reply:
I have absolutely no clue LOL. Fixed.

Reply:
I even checked the diff before committing. I was wondering why this was showing up in the status dialog.